app/uk/gov/hmrc/thirdpartyapplication/repository/ApplicationRepository.scala

```diff
@@ -332,6 +344,71 @@ class ApplicationRepository @Inject() (mongo: MongoComponent)(implicit val ec: E
     collection.find(query).toFuture()
   }
 
+  def getSubscriptionsForDeveloper(userId: UserId): Future[Set[ApiIdentifier]] = {
```

Comment:
What is the reason for having this in the ApplicationRepository rather than the SubscriptionRepository?

Reply:
Secondly, if this method is supposed to be in AppRepo, can it be deleted from SubsRepo, as that copy is no longer used (except in tests).

Reply:
1. it's in app repo now because it needs to start the query against application and then lookup subscription whereas before it used to query against subscription and then lookup application. The joy of "joins" in a non relational db.

2. I thought I had done so, clearly not added a subsequent change after the tests were fixed.  Thanks for spotting this.

Reply:
Looks like I completely missed this.  I blame it on end of day euphoria ;-)